docs/standard/datetime/datetime-support-in-system-text-json.md

```diff
@@ -0,0 +1,218 @@
+
+2. `date-time (5.4.2.1)`
+	1. `YYYY-MM-DDThh:mm:ssZ`
+	Used to format a <xref:System.DateTime> or <xref:System.DateTimeOffset> without fractional seconds and of kind <xref:System.DateTimeKind.Utc>
+
+	2. `YYYY-MM-DDThh:mm:ss.sZ`
+	Used to format a <xref:System.DateTime> or <xref:System.DateTimeOffset> with fractional seconds and of kind <xref:System.DateTimeKind.Utc>
+	
+	3. `YYYY-MM-DDThh:mm:ss("+" / "-")hh":"mm`
+	Used to format a <xref:System.DateTime> or <xref:System.DateTimeOffset> without fractional seconds and of kind <xref:System.DateTimeKind.Local>
+
+	4. `YYYY-MM-DDThh:mm:ss.s("+" / "-")hh":"mm`
+	Used to format a <xref:System.DateTime> or <xref:System.DateTimeOffset> with fractional seconds and of kind <xref:System.DateTimeKind.Local>
+
+## Custom support for date and times in `JsonSerializer`
+
+If you want custom parsing or formatting at the serializer level, you can implement [custom converters](https://github.com/dotnet/corefx/issues/36639). Here are a few examples:
+
+1. Using `DateTime(Offset).Parse` and `DateTime(Offset).ToString`. This allows you to use .NET's most comprehensive effort at parsing various date-time formats.
+This is significantly less performant than using the serializer's native parsing implementation.
+
+```C#
+using System.Text.Json.Serialization;
+
+namespace Example1
+{
+    public class DateTimeConverter : JsonConverter<DateTime>
+    {
+        public override DateTime Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            if (reader.TokenType != JsonTokenType.String)
+            {
+                throw new JsonException();
+            }
+
+            reader.Read();
+            return DateTime.Parse(reader.GetString());
+        }
+
+        public override void Write(Utf8JsonWriter writer, DateTime value, JsonSerializerOptions options)
+        {
+            writer.WriteStringValue(value.ToString());
+        }
+    }
+
+    public class DateTimeOffsetConverter : JsonConverter<DateTimeOffset>
+    {
+        public override DateTimeOffset Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            if (reader.TokenType != JsonTokenType.String)
+            {
+                throw new JsonException();
+            }
+
+            reader.Read();
+            return DateTimeOffset.Parse(reader.GetString());
+        }
+
+        public override void Write(Utf8JsonWriter writer, DateTimeOffset value, JsonSerializerOptions options)
+        {
+            writer.WriteStringValue(value.ToString());
+        }
+    }
+}
+```
+
+2. Using `UTF8Parser` and `UTF8Formatter`. This allows you to use fast parsing and formatting methods for Ut8 date-time data that is compliant with one of the [Standard Date and Time Format Strings](https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings).
+This is much faster than Example 1.
```

Comment:
If it's much faster than Example 1 what would lead you to choose one over the other?

Reply:
I've added some guidance.